### PR TITLE
[WIP] Review/proofread

### PR DESCRIPTION
**Pull request by simonbray**
Not complete. Still need to check the python scripts as well.
* [ ] Flake8 passes (`flake8 . --exclude=.venv,.build,planemo_test_env,build --ignore=E501,F403,E402,F999,F405,E712`)
* [ ] Local tests pass (`py.test hicexplorer --doctest-modules`)

### CODE REVIEW

**Comment by joachimwolff** [comment]
Thanks a lot Simon!

File: hicexplorer/chicDifferentialTest.py [hunk 1]
```diff
@@ -21,8 +21,8 @@ def parse_arguments(args=None):
     parser = argparse.ArgumentParser(add_help=False,
                                      formatter_class=argparse.RawDescriptionHelpFormatter,
                                      description="""
-chicDifferentialTest tests if two locations under consideration of the reference point have a different interaction count. For this either Fisher's test or chi2 contingency test can be used.
-The files that are accepted for this test can be created with `chicAggregateStatistic`. H0 is assuming the interactions are not different. Therefore the differential interaction counts are all where H0 was rejected.
+chicDifferentialTest tests if two locations under consideration of the reference point have a different interaction count. For this either Fisher's test or the chi2 contingency test can be used.
+The files that are accepted for this test can be created with `chicAggregateStatistic`. H0 assumes the interactions are not different. Therefore the differential interaction counts are all where H0 was rejected.
```
Comment (simonbray) [inline hicexplorer/chicDifferentialTest.py]: cannot judge this but I guess your users understand what H0 means? For me it's not clear.

File: hicexplorer/chicViewpoint.py [hunk 2]
```diff
@@ -46,34 +45,34 @@ def parse_arguments(args=None):
                                 nargs='+')
 
     parserRequired.add_argument('--range',
-                                help='Defines the region upstream and downstream of a reference point which should be considered in the analysis. Please have in mind to use the same fixate range setting as it was '
-                                'for the background model computation and that distances of the range larger as the fixate range use the background model of those.'
+                                help='Defines the region upstream and downstream of a reference point which should be considered in the analysis. Please remember to use the same fixate range setting as '
+                                'for the background model computation and that distances of the range larger than the fixate range use the background model of those.'
```
Comment (simonbray) [inline hicexplorer/chicViewpoint.py]: can you review this line please? Some words seem to be missing, I don't understand what 'those' means.

**Comment by simonbray** [comment]
@joachimwolff I think this was everything you wanted checking, right?

You should check my corrections and make sure I didn't accidentally change the meaning of any sentences.

**Pull request merged**